usecases/config/runtimeconfig.go

```diff
@@ -36,6 +37,16 @@ type WeaviateRuntimeConfig struct {
 	QuerySlowLogEnabled            *runtime.DynamicValue[bool]          `json:"query_slow_log_enabled" yaml:"query_slow_log_enabled"`
 	QuerySlowLogThreshold          *runtime.DynamicValue[time.Duration] `json:"query_slow_log_threshold" yaml:"query_slow_log_threshold"`
 	InvertedSorterDisabled         *runtime.DynamicValue[bool]          `json:"inverted_sorter_disabled" yaml:"inverted_sorter_disabled"`
+
+	// Experimental configs. Will be removed in the future.
+	// OIDCEnabled           *runtime.DynamicValue[bool]     `json:"exp_oidc_enabled" yaml:"exp_oidc_enabled"`
```

Comment:
why is this commented out?

Reply:
Yep, if intended it's all good then - do you want to keep the commented out value here? If yes could you add a little comment why it is commented out? Was just unsure if it was an oversight or not

Reply:
I removed it :)